openassessment/templates/openassessmentblock/grade/oa_grade_closed.html

```diff
@@ -1,16 +0,0 @@
-{% load i18n %}
-{% spaceless %}
```

Comment:
Note to reviewers: this template was unused (I noticed the text didn't match the name of the file).